File: week1/AndYouAre.cpp
/**
 * A simple flow control example program.
 * 
 * @author: Mark Orimoloye
 */
 
#include <iostream>
#include <string>
using namespace std;

int main()
{
    string name;
	cout << "Hello, and you are? ";
	cin >> name;
	
	if (name == "Mark")
	{
	    cout << "Hello " + name + " welcome to your class.";
	}
	else if (name == "Matthew")
	{
	    cout << " Hello Professor " + name + " Brown~!";
	}
	else 
	{
	    cout << " Hello " + name + ", are you in the right class?";
	}
}

File: week1/AndYouAre.java
/**
 * A simple flow control example program.
 * 
 * @author: Mark Orimoloye
 */

import java.util.Scanner;

public class AndYouAre 
{
	public static void main(String[] args) 
	{
    	System.out.println("Hello, and you are?");
    	Scanner in = new Scanner(System.in);
    	String s = in.nextLine();
    	
    	// An example of branching with the If Statement
    	if(s.equals("Mark"))
    	{
    		System.out.println("Hi Mark~!");
    	}
    	else if (s.equals("Matthew")) 
    	{
    		System.out.println("Welcome Professor Brown~!");
    	}
    	else 
    	{
    		System.out.println("Hello, not sure who you are.");
    	}
    	in.close();
	}
}